### DISCUSSION TITLE
是否有计划在后续的更新中暴露label元素

### DISCUSSION THREAD

**Discussion by BDN233** [antvis/G6#6102] [Q&A]
如果可以获取label元素并可以获取其Keyshape，我认为这样对于操作修改文字标注的位置的便捷性将大大提高，并且可以完成例如
检测文字是否碰撞等操作。

**Comment by Aarebecca** [comment]
@BDN233 可以通过元素示例上的 `getShape` 方法获取对应的子元素，如 `element.getShape('key')` 获取主图形，`element.getShape('label')` 获取标签
